sentinel-core/src/core/circuitbreaker/breaker/mod.rs

```diff
@@ -101,6 +101,11 @@ pub trait StateChangeListener: Sync + Send {
     /// Argument rule is copy from circuit breaker's rule, any changes of rule don't take effect for circuit breaker
     /// Copying rule has a performance penalty and avoids invalid listeners as much as possible
     fn on_transform_to_half_open(&self, prev: State, rule: Arc<Rule>);
+
+    /// `on_circuit_breaker_drop` is triggered when circuit breaker is drop.
+    /// Argument rule is copy from circuit breaker's rule, any changes of rule don't take effect for circuit breaker
+    /// Copying rule has a performance penalty and avoids invalid listeners as much as possible
+    fn on_circuit_breaker_drop(&self, prev: State, rule: Arc<Rule>);
```

Comment:
I think an empty default trait impl would be helpful if the developer wants to omit it. The same is true to other interfaces in the `StateChangeListener`,  could you help add them? 

Or I think these [circuit_breaker examples](https://github.com/sentinel-group/sentinel-rust/tree/main/examples/rules/circuit_breaker) the [prometheus example](https://github.com/sentinel-group/sentinel-rust/blob/main/examples/exporter/prometheus/prometheus.rs) won't compile. We didn't check these examples in the CI.

Reply:
resolved by #134